docs/custom-queries.md

```diff
@@ -0,0 +1,25 @@
+# Making Custom Calls to Graph
+
+The Graph SDK attempts to enable all available scenarios through Microsoft Graph. There are times, however, through errors or custom Graph functionality, that makes calling the desired endpoint is not possible through the provided requests and builders.
+
+## Creating a custom request
+You can extend BaseRequest to create a custom request:
+
+```Java
+public class CustomRequest extends BaseRequest {
+
+    public CustomRequest(final String requestUrl, final IBaseClient client, final java.util.List<Option> requestOptions) {
+        super(requestUrl, client, requestOptions, Void.class);
+    }
+
+    public String get() throws ClientException {
+        return send(HttpMethod.GET, null);
+    }
+}
+```
+Then you can instantiate a new request to use to call Graph:
+
+```Java
+CustomRequest request = new CustomRequest("https://graph.microsoft.com/v1.0/custom", graphServiceClient, requestOptions);
```

Comment:
Are there any limitations that we should mention here? Can any URL be passed in or does it have to be a known graph deployment (main/china cloud/etc)? Can you pass in query parameters via the URL or do they need to be passed in via requestOptions?